source/ceylon/file/internal/temporaryDirectory.ceylon

```diff
@@ -0,0 +1,20 @@
+import ceylon.file {
+    Directory
+}
+
+shared Directory temporaryDirectory() {
```

Comment:
Ok, I'll change it (I made it a function to highlight it's laziness as opposed to the `shared` `temporaryDirectory`, but a comment would be better.)

@gavinking on this topic... `temporaryDirectory` is eager (well, calculated on first use) to match Java's behavior. But, we could make it lazy to allow apps to redefine `java.io.tmpdir`. I'm not sure what their motivation was to use a private static to hold the value. It could be security, thread safety, dislike for mutable global state, or hatred of users.